basis/functors/functors.factor

```diff
@@ -159,16 +153,14 @@ DEFER: ;FUNCTOR delimiter
     [
         building get use-words
         (parse-bindings)
-        building get unuse-words
     ] with-bindings ;
 
 : parse-functor-body ( -- form )
-    push-functor-words
+    functor-words use-words
```

Comment:
This is outside the scope of the ``with-lambda-scope`` so shouldn't we still ``unuse-words``?